jgiven-gradle-plugin/build.gradle

```diff
@@ -15,26 +15,20 @@ dependencies {
 
 }
 
-gradlePlugin {
-    plugins {
-        jgivenPlugin {
-            id = "com.tngtech.jgiven.gradle-plugin"
-            implementationClass = "com.tngtech.jgiven.gradle.JGivenPlugin"
-        }
-    }
-}
-
 pluginBundle {
     website = 'http://jgiven.org/'
     vcsUrl = 'https://github.com/TNG/JGiven.git'
-    description = 'JGiven Gradle plugin'
     tags = ['jgiven', 'testing','bdd','junit','testng']
     //version = '1.0.0-RC1.4'
+}
 
+gradlePlugin {
     plugins {
         jgivenPlugin {
-            id = gradlePlugin.plugins.jgivenPlugin.id
+            id = "com.tngtech.jgiven.gradle-plugin"
             displayName = 'Gradle JGiven Plugin'
+            implementationClass = "com.tngtech.jgiven.gradle.JGivenPlugin"
         }
     }
 }
+
```

Comment:
according to https://guides.gradle.org/publishing-plugins-to-gradle-plugin-portal/ the two ways to add information are equivalent